src/components/Nav/Nav.Props.ts

```diff
@@ -121,6 +136,12 @@ export interface INavLink {
   target?: string;
 
   /**
+   * Point to the parent node key.  This is used in EditNav when move node from sublink to
+   *   parent lin vs vers.
```

Comment:
ok updated this field should be parent node key id value.  Used by server during moving nodes around.